_data/tool_and_resource_list.yml

```
@@ -1749,12 +1749,12 @@
   id: research-data-management-organiser
   name: Research Data Management Organiser
   url: https://rdmorganiser.github.io/en
-- description: Data management platform for automated loading, storage, linkage and provision of data sets
+- description: Research Data Management Platform (RDMP) at Health Informatics Centre, University of Dundee for automated loading, storage, linkage and provision of data sets
   id: research-data-management-platform
-  name: Research Data Management Platform (RDMP)
+  name: RDMP
```

```suggestion
  name: Research Data Management Platform (RDMP)
```

@bianchini88 I personally like the combination full name + acronym, like we have been doing for many other tools. THe tool id itself is better the acronym imo